### ISSUE TITLE
sitewide, forms 508-defect-3 [FOCUS MANAGEMENT]: Focus outline SHOULD be removed from non-linked heading

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9218]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
! Team affected: VSA Caregiver; Project: 10-10CG MVP
```

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Sitewide issue documented in https://github.com/department-of-veterans-affairs/va.gov-team/issues/9116.

Focus outline **should** not be applied to non-interactive focusable content, as a pattern on VA.gov.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to identify interactive focusable items by distinctive focus styles.

## Environment

Operating System: all
Browser: all
Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/caregivers/ in browser
2. Navigate through to the review-and-submit screen
3. Move through the process flow, noting that the focus outline appears on the non-linked headings as shown in the screenshot

## Possible Fixes (optional)

See [#457](https://github.com/department-of-veterans-affairs/va.gov-team/issues/457)

## WCAG or Vendor Guidance (optional)

* [W3C Understanding WCAG 2.0 - Consistent Identification SC 3.2.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html)

## Screenshots or Trace Logs

![Screen Shot 2020-05-18 at 4 09 11 PM](https://user-images.githubusercontent.com/57469/82255039-f0bb5200-9921-11ea-8e47-6eb6e0b44449.png)

**Comment by k80bowman** [comment]
The original link provided no longer works, but I checked this on https://staging.va.gov/family-member-benefits/apply-for-caregiver-assistance-form-10-10cg/vet-1# and I did not see the focus outline appearing on the non-linked heading. See video below. Closing as I believe this has been resolved.

https://images.zenhubusercontent.com/133843125/a4a1f873-b8df-4c28-af32-54d82d147712/screen_recording_2022_01_26_at_9_18_32_am.mov
